app/assets/stylesheets/pages/login.scss

```diff
@@ -0,0 +1,32 @@
+///login page styleing
+
+#login {
+  background-image: url("BackgroundImage.jpeg");
+  background-size: cover;
+    .card {
+      border-radius: 12px;
+      margin-top: 30vw;
+      margin-left: 5%;
+      margin-right: 5%;
+      .fa-heart-o{
+        color: white;
+      }
+      .facebook-square{
+        margin: 0;
+      }
+      .btn-warning{
+        margin-top: 2%;
+        margin-bottom: 2%;
```

Comment:
If we are trying to go responsive, we should really standardize our scale of where we are spacing things out. but for now we can let it slide

Reply:
Alternative is to rely on the bootstrap framework to do the responsive design for us and avoid super-specific spacing unless necessary

Reply:
so can you elaborate when you say standardize our scaling. I really didn't need to use %. How would we use bootstrap here for spacing?